examples/vision/retinanet.py

```diff
@@ -0,0 +1,700 @@
+"""
+Title: Object Detection with RetinaNet
+Author: [Srihari Humbarwadi](https://twitter.com/srihari_rh)
+Date created: 2020/05/17
+Last modified: 2020/06/22
+Description: Implementation of Focal Loss for Dense Object Detection.
+"""
+import os
+import re
+import zipfile
+
+import numpy as np
+import tensorflow as tf
+from tensorflow import keras
+
+import matplotlib.pyplot as plt
+import tensorflow_datasets as tfds
+
+"""
+## Download subset of COCO2017 dataset
```

Comment:
Start with an `Introduction` section that explains the overall architecture you use and the different components you will need to implement it. This example should serve as an introduce to object detection.